catroid/src/main/java/org/catrobat/catroid/ui/recyclerview/fragment/CatblocksScriptFragment.kt

```diff
@@ -413,6 +414,9 @@ class CatblocksScriptFragment(
 
             return Gson().toJson(brickCategoryInfos)
         }
+
+        @JavascriptInterface
+        fun isAdvancedMode(): Boolean = advancedMode
```

Comment:
The setting is not read, so it's always false for Catblocks, is this intended at this point?